app/src/main/res/layout/activity_request_detail.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
+<ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
+<androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
it should be wrap content if you want to scroll, you think?

Reply:
I'm not sure but I think the height should be wrap_content. match_parent for the width is fine.

Reply:
Agree!

Reply:
> Agree!

changes made :)